packages/app/src/messages/components/thread-details-messages.component.tsx

```diff
@@ -14,25 +20,118 @@ import {
   MessageType,
   Receiver,
 } from "App/messages/reducers/messages.interface"
+import NewMessageBadge from "App/messages/components/new-message-badge.component"
+import { Notification } from "App/notification/types"
 
 interface Properties {
   messages: Message[]
   receiver?: Receiver
+  messageLayoutNotifications?: Notification[]
+  removeLayoutNotification?: (notificationId: string) => void
 }
 
 const ThreadDetailsMessages: FunctionComponent<Properties> = ({
   messages,
   receiver,
+  messageLayoutNotifications,
+  removeLayoutNotification,
 }) => {
-  const ref = useRef(null);
+  const newRef = useRef<HTMLDivElement>(null)
+  const ref = useRef<HTMLDivElement>(null)
+  const [notifications, setNotifications] = useState<
+    Notification[] | undefined
+  >()
```

Comment:
Isn't be easier to handle logic  just with array type? 🤔 
```suggestion
  const [notifications, setNotifications] = useState<
    Notification[]
  >([])
```

Reply:
no complete 😉 ➡️ https://github.com/mudita/mudita-center/pull/712/files#r869989545

Reply:
You're right! Fixed on [bd6a8e1](https://github.com/mudita/mudita-center/pull/712/commits/bd6a8e1d55ae99d73ce4d756240c62a1f3362711)